src/components/data-collection-page/DataCollectionPage.tsx

```diff
@@ -71,33 +73,38 @@ export default function DataCollectionPage(props: IProps) {
 			return;
 		}
 
-		props.handleDataCollection({
-			scoutTeamNumber: scoutTeamNumber,
+		const submitMatchData = () => ({
```

Comment:
This is creating a function that return match data, but doesn't actually do anything with it. It's a simple fix though.

Change line 76 to `matchData: IMatch = {`
Line 84 can just be `objectives: [`
And line 105 should be `props.submitMatchData(matchData);`